src/app/services/ListItem.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import Icon from '../../components/icon';
+
+function ListItem({ title, progress, onClick }) {
+  return (
+    <div className="ListItem py-4 border-top border-bottom" style={{ cursor: 'pointer' }}>
```

Comment:
I assume these will become actual links in a later PR, right?

Reply:
Fixed in this one 👍